examples/plot_bcic_iv_4_ecog_trial.py

```diff
@@ -0,0 +1,287 @@
+"""
+Trialwise Decoding on BCIC IV 4 ECoG Dataset
+========================================
+
+This tutorial shows you how to train and test deep learning models with
+Braindecode with ECoG
+a classical EEG setting: you have trials of data with
+labels (e.g., Right Hand, Left Hand, etc.).
+
+"""
+
+######################################################################
+# Loading and preprocessing the dataset
+# -------------------------------------
+#
+
+
+######################################################################
+# Loading
+# ~~~~~~~
+#
+
+
+######################################################################
+# First, we load the data. In this tutorial, we use the functionality of
+# braindecode to load datasets through
+# `MOABB <https://github.com/NeuroTechX/moabb>`__ to load the BCI
+# Competition IV 2a data.
+#
+# .. note::
+#    To load your own datasets either via mne or from
+#    preprocessed X/y numpy arrays, see `MNE Dataset
+#    Tutorial <./plot_mne_dataset_example.html>`__ and `Numpy Dataset
+#    Tutorial <./plot_custom_dataset_example.html>`__.
+#
+from sklearn.metrics import r2_score
+
+DATASET_PATH = '/home/maciej/projects/braindecode/BCICIV_4_mat'
```

Comment:
I assume you may have to mock the dataset like Lukas did in https://github.com/braindecode/braindecode/pull/276 for this to run on CI eventually? Or what is your plan?

Reply:
Yeah, I will try to prepare something similar! :)

Reply:
This one is still to be done. Mocking and testing BCICompetitionDataset4